MAUI/Effects-View/Getting-Started.md

```diff
@@ -7,19 +7,35 @@ control: Effects View
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI Effects View (SfEffectsView)
+# Getting Started with .NET MAUI Effects View
+This section guides you through setting up and configuring a
+[Effects View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html?tabs=tabid-1)
+in your .NET MAUI application. Follow the steps below to add [Effects View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html?tabs=tabid-1)
+to your project.
 
-This section explains the steps required to configure the .NET MAUI Effects View control and customize its elements.
-
-To get start quickly with our .NET MAUI Effects View, you can check the below video.
+To get start quickly with our .NET MAUI EffectsView, you can check the below video.
 
 {% youtube "https://www.youtube.com/watch?v=IOieyRMCuAo" %}
 
-## Adding a SfEffectsView reference
+## Prerequisites
+Before proceeding, ensure the following are set up:
+
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code).
+
+## Step 1: Create a New MAUI Project
 
-Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [SfEffectsView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html?tabs=tabid-1) to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and then install it.
+1. Launch Visual Studio or VS Code.
+2. Navigate to **File > New > Project,** then select the .NET MAUI App template.
+3. Name the project and choose a location, then click Create.
 
-## Handler registration 
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
+
+1.  In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+2.  Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and install the latest version.
+3.  Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+## Step 3: Handler registration 
 
  In the MauiProgram.cs file, register the handler for Syncfusion core.
```

Comment:
Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.

Reply:
Yes, I have changed.